opentelemetry-api/src/opentelemetry/propagators/textmap.py

```diff
@@ -80,6 +95,25 @@ def keys(self, carrier: typing.Dict[str, CarrierValT]) -> typing.List[str]:
         return list(carrier.keys())
 
 
+default_getter = DictGetter()
+
+
+class DictSetter(Setter):
+    def set(
+        self, carrier: typing.Dict[str, CarrierValT], key: str, value: str
```

Comment:
`typing.MutableMapping` would be more appropriate here (or can make a custom minimal protocol for it)